apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -273,26 +273,27 @@ export class IENApplicantUtilService {
   async updateLatestStatusOnApplicant(mappedApplicantList: string[]): Promise<void> {
     try {
       // update applicant with the latest status
-      const idsToUpdate = `'${mappedApplicantList.join("','")}'`;
-      const queryToUpdate = `
-        UPDATE ien_applicants
-        SET status_id = (
-          SELECT status_id
-          FROM ien_applicant_status_audit asa
-          WHERE asa.applicant_id=ien_applicants.id
-          ORDER BY asa.start_date
-          DESC limit 1
-        )
-        WHERE ien_applicants.id IN (${idsToUpdate})`;
-      const updatedApplicants = await getManager().query(queryToUpdate);
-      this.logger.log({ updatedApplicants });
+      const subQuery = this.ienapplicantStatusAuditRepository
+        .createQueryBuilder('audit')
+        .select(['status_id'])
+        .where(`audit.applicant_id = applicant.id`)
+        .orderBy('audit.start_date', 'DESC')
+        .limit(1);
+      const result = await this.ienapplicantRepository
+        .createQueryBuilder('applicant')
+        .update({
+          status: () => '(' + subQuery + ')',
+        })
+        .where({ id: In(mappedApplicantList) })
+        .execute();
+      this.logger.log(`applicants status updated: ${result.affected}`);
```

Comment:
@dbayly-freshworks 
I changed the raw query to query builder and restored logging